utils/warnings.ml

```diff
@@ -220,8 +220,14 @@ let is_active x = (!current).active.(number x);;
 let is_error x = (!current).error.(number x);;
 
 let parse_opt error active flags s =
-  let set i = flags.(i) <- true in
-  let clear i = flags.(i) <- false in
+  let set i =
+    let i = if i = 25 then 8 else i in
+    flags.(i) <- true
+  in
+  let clear i =
+    let i = if i = 25 then 8 else i in
+    flags.(i) <- false
```

Comment:
I prefer not to introduce such an asymmetrical hack to preserve a scenario that seems a bit crazy (enabling warning 25 but not 8).